### DISCUSSION TITLE
隐私计算论文合集「联邦学习系列」第2期

### DISCUSSION THREAD

**Discussion by halosecret00** [secretflow/secretflow#576] [General]
![图片](https://img-blog.csdnimg.cn/img_convert/16e835c4039bd51d343f368d20d7a1b7.png)  

**前言：**

隐语awesome-PETs（PETs即Privacy-Enhancing Technologies ，隐私增强技术）精选业内优秀论文，按技术类型进行整理分类，旨在为隐私计算领域的学习研究者提供一个高质量的学习交流社区。awesome-pets包含：安全多方计算(MPC)、零知识证明(ZKP)、联邦学习(FL)、差分隐私(DP)、可信执行环境(TEE)、隐私求交(PSI)等系列主题论文！

继上期[**《多方安全计算》系列论文推荐活动**小伙伴们参与热烈，社区收到了不少Paper留言。

本期继续带来联邦学习 (FL）系列论文推荐，更多主题Paper持续更新中ing～欢迎收藏项目。https://github.com/secretflow/secretflow/blob/main/docs/awesome-pets/awesome-pets.md

提交PR，推荐“联邦学习”主题论文，私信隐语小助手SecretFlow01，可参与抽奖活动！（中奖率超高呦～奖品见下文）且推荐论文被合并的贡献者，还将在隐语官方repo中进行@，以肯定及感谢您在awesome-pets项目中的贡献成果。https://github.com/secretflow/secretflow/pulls

**联邦学习系列论文**

**1、Survey**

General

*   Federated machine learning: Concept and applications  
    
*   Federated Learning in Mobile Edge Networks: A Comprehensive Survey
    
*   Advances and Open Problems in Federated Learning
    
*   Federated Learning: Challenges, Methods, and Future Directions
    

Security

*   A survey on security and privacy of federated learning
    
*   Threats to Federated Learning: A Survey
    
*   Vulnerabilities in Federated Learning
    

由于篇幅原因，**还有更多论文**未能一一列举，请访问下方链接收藏！https://github.com/secretflow/secretflow/blob/main/docs/awesome-pets/papers/applications/ppml/fl/fl.md

**2、Datasets**

*   LEAF: A Benchmark for Federated Settings HomePage
    
*   UniMiB SHAR: a new dataset for human activity recognition using acceleration data from smartphones HomePage
    
*   The OARF Benchmark Suite: Characterization and Implications for Federated Learning Systems
    
*   Evaluation Framework For Large-scale Federated Learning
    
*   (\*) PrivacyFL: A simulator for privacy-preserving and secure federated learning. MIT CSAIL.
    
*   Revocable Federated Learning: A Benchmark of Federated Forest
    

  

**3、Efficiency**

Quantization

*   Communication-Efficient Distributed Learning via Lazily Aggregated Quantized Gradients
    
*   Lazily Aggregated Quantized Gradient Innovation for Communication-Efficient Federated Learning
    
*   Communication Efficient Federated Learning with Adaptive Quantization
    
*   QSGD: Communication-Efficient SGD via Gradient Quantization and Encoding
    
*   DEED: A General Quantization Scheme for Communication Efficiency in Bits
    

  

**4、Effectiveness**

Model Aggregation

*   FedAvg: Communication-Efficient Learning of Deep Networks from Decentralized Data
    
*   LAG: Lazily Aggregated Gradient for Communication-Efficient Distributed Learning
    
*   Federated Learning with Matched Averaging
    
*   Federated Learning of a Mixture of Global and Local Models
    
*   Faster On-Device Training Using New Federated Momentum Algorithm
    
*   FedDANE: A Federated Newton-Type Method
    
*   SCAFFOLD: Stochastic Controlled Averaging for Federated Learning
    

**5、Incentive**

Contribution Evaluation

*   Data Shapley: Equitable Valuation of Data for Machine Learning
    
*   A principled approach to data valuation for federated learning
    
*   Measure contribution of participants in federated learning
    
*   GTG-Shapley: Efficient and Accurate Participant Contribution Evaluation in Federated Learning
    
*   Profit allocation for federated learning
    
*   Fedcoin: A peer-to-peer payment system for federated learning
    

Profit Allocation

*   Hierarchical Incentive Mechanism Design for Federated Machine Learning in Mobile networks
    
*   FAIR: Quality-Aware Federated Learning with Precise User Incentive and Model Aggregation
    
*   Incentive Mechanism for Horizontal Federated Learning Based on Reputation and Reverse Auction
    

**6、Vertical FL**

*   SecureBoost: A Lossless Federated Learning Framework
    
*   Parallel Distributed Logistic Regression for Vertical Federated Learning without Third-Party Coordinator
    
*   Entity Resolution and Federated Learning get a Federated Resolution.
    
*   Multi-Participant Multi-Class Vertical Federated Learning
    
*   A Communication-Efficient Collaborative Learning Framework for Distributed Features
    
*   Asymmetrical Vertical Federated Learning
    

**7、Boosting**

*   Practical Federated Gradient Boosting Decision Trees
    
*   Secureboost: A lossless federated learning framework
    
*   Large-scale Secure XGB for Vertical Federated Learning
    

**8、Application**

Natural language Processing

*   Federated pretraining and fine tuning of BERT using clinical notes from multiple silos
    
*   Federated Learning for Mobile Keyboard Prediction
    
*   Federated Learning for Keyword Spotting
    
*   generative sequence models (e.g., language models)
    
*   Federated User Representation Learning
    
*   Two-stage Federated Phenotyping and Patient Representation Learning
    

**由于篇幅原因，**还有更多论文**未能一一列举，请访问下方链接收藏！**

https://github.com/secretflow/secretflow/blob/main/docs/awesome-pets/papers/applications/ppml/fl/fl.md

**Comment by Mingbo-Lee** [comment]
冒昧说一句，标题“私计算论文合集「联邦学习系列」第2期 ”缺少一个“隐”字，建议更改一下。

**Reply by halosecret00** [reply]
> 冒昧说一句，标题“私计算论文合集「联邦学习系列」第2期 ”缺少一个“隐”字，建议更改一下。

好的，收到。
